src/main/java/com/artipie/rpm/Rpm.java

```diff
@@ -81,25 +87,32 @@
      */
     private final NamingPolicy naming;
 
+    /**
+     * Flag for filelists file generation.
+     */
+    private final boolean filelistsgen;
+
     /**
      * Ctor.
      * @param stg Storage
      */
     public Rpm(final Storage stg) {
-        this(stg, NamingPolicy.DEFAULT);
+        this(stg, NamingPolicy.DEFAULT, true);
     }
 
     /**
      * Ctor.
      * @param stg The storage
      * @param naming RPM files naming policy
+     * @param filelistsgen Flag to generate the filelist.
      */
-    public Rpm(final Storage stg, final NamingPolicy naming) {
+    public Rpm(final Storage stg, final NamingPolicy naming, final boolean filelistsgen) {
```

Comment:
@paulodamaso Do we have an alternative to a boolean parameter? Using this class is not user friendly:
` new Rpm(storage, NamingPolicy.DEFAULT, false)` How about using `factory` classes or wrapping the boolean to an object?